hivemind/p2p/p2p_daemon.py

```diff
@@ -83,30 +80,27 @@ async def create(cls, *args, quic: bool = True, tls: bool = True, conn_manager:
         :param nat_port_map: Enables NAT port mapping
         :param auto_nat: Enables the AutoNAT service
         :param bootstrap: Connects to bootstrap peers and bootstraps the dht if enabled
-        :param bootstrap_peers: List of bootstrap peers; defaults to the IPFS DHT peers
-        :param use_global_ipfs: Bootstrap to global ipfs (works only if bootstrap=True and bootstrap_peers=None)
-        :param host_port: port for p2p network
+        :param bootstrap_peers: List of bootstrap peers
+        :param use_ipfs: Bootstrap to IPFS (works only if bootstrap=True and bootstrap_peers=None)
+        :param p2p_port: port for p2p network
```

Comment:
nit: i would argue that "port for p2p network" is still a bit misleading, aka "how does an entire network have a single port?"

```suggestion
        :param p2p_port: port for external connections from other p2p instances
```

Reply:
I've used your comment and renamed it to `external_port` :)